src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -671,7 +671,7 @@ public void deleteFeedbackResponsesForQuestionAndCascade(
             emails.add(response.giverEmail);
         }
 
-        if(!hasResponseRateCheck){
+        if(!hasResponseRateUpdate){
```

Comment:
@unyoungwax, not related to your changes, but since you touched this line, let's fix the indentation here :P